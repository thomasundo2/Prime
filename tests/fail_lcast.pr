int main()
{
    lint x;
    x = tolint(12l);
}