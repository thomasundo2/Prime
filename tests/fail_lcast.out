Fatal error: exception Failure("illegal argument lint expected int in 12")